transport/http_util.go

```diff
@@ -454,7 +454,7 @@ func encodeGrpcMessage(msg string) string {
 	lenMsg := len(msg)
 	for i := 0; i < lenMsg; i++ {
 		c := msg[i]
-		if !(c >= spaceByte && c < tildaByte && c != percentByte) {
+		if !(c >= spaceByte && c <= tildaByte && c != percentByte) {
```

Comment:
optional: In English `~` is commonly spelled `tilde`

Reply:
😂fixed